CMakeLists.txt

```diff
@@ -39,24 +38,28 @@ set(BAREFLANK_INSTALL_TEST_PREFIX ${CMAKE_BINARY_DIR}/bfprefix_test)
 if(NOT WIN32)
 
     find_program(PYTHON_BIN python)
+    set(PYTHON_BIN ${PYTHON_BIN} CACHE INTERNAL "")
```

Comment:
If you declare a cmake variable as "CACHE INTERNAL", it will remove it as a visible configuration option from cmake-gui and ccmake (the cmake curses gui).  I marked a lot of the variables that currently don't make sense as "configurable" CACHE INTERNAL. This particular variable will probably become configurable once I migrate PYTHON_BIN into an appropriate location in a toolchain or configuration file. That way you can configure a specific version of python if you so desire.

Reply:
Ah, thanks, yeah I don’t use cmake-gui, but I am glad that you are so that this is being tested.